.gitignore

```diff
@@ -1,7 +1,8 @@
 node_modules
-## Yarn
 package-lock.json
-node_modules/
+temp
```

Comment:
That is an interesting way of naming a temporary folder :D, I usually set `tmp` as a name, as it is a unix standard. Nonetheless, you can ignore this comment. I am curious though to know why you used that `temp` folder.

Reply:
Normally I use `tmp` folders to test or create something local that I want to make sure that I will not commit accidentally. It is more a way-of-work for me than a need for the application.

Directory renamed 🙂